.github/workflows/test.yml

```diff
@@ -0,0 +1,27 @@
+name: Run Tests
+
+on: 
+  push:
+  pull_request:
+
+concurrency:
+  group: "test"
+  cancel-in-progress: false
+
+jobs:
+  test:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23]
```

Comment:
We can probably drop a lot of these older ones.

Reply:
From which version do you want to start testing?

Reply:
now use lts versions